tests/Phinx/Migration/Manager/EnvironmentTest.php

```diff
@@ -11,6 +11,7 @@ class PDOMock extends \PDO
 {
     public function __construct()
     {
+        parent::__construct('sqlite::memory:');
```

Comment:
I think the problem is here. You are forcing all tests to use `sqlite::memory`, regardless of what the tests is setup to do.